WordPress/gutenberg
# Base Styles: Extract long-content-fade to its own file so it can be imported and reused.

## What?

Extract the `long-content-fade` mixin to it's own file.

## Why?

So that it can be easily reused by importing when needed. Whenever we need to use it we don't have to include the other mixins in the `_mixins.scss` which might cause collision in other codebases.
## How?

I'm moving the `long-content-fade` mixin from the `_mixins.scss` file into it's own file so it can be imported and reused easily.
## Testing Instructions




1. Checkout his branch to your local repo
2. Verify `_long-content-fade.scss`  can be imported without build errors.
3. In editor add a search block and remove buttons and labels. Preview the post:
       - in html remove search block `.wp-block-search__inside-wrapper` 
       - add class 
```css
.search-component__input-fade {
	display: flex;
	align-items: center;
	border-radius: 0;
	flex: 1 1 auto;
	height: 100%;
	position: relative;
	&::before {
		@include long-content-fade( $direction: right, $size: 70px, $z-index: 2, $color: #fff, $edge: 1px );
		border-radius: inherit;
	}
}
```
5. Verify the long-content-fade is still applied as shown below:
![image](https://user-images.githubusercontent.com/47489215/208499614-5ed51fc8-ac37-4cb5-b795-e43a119f01d2.png)
Notice fading edges where the text overflows.

### Testing Instructions for Keyboard


## Screenshots or screencast
